package-lock.json

```diff
@@ -0,0 +1,4328 @@
+{
```

Comment:
Are we using npm ci for the collectors too?

Reply:
I added the package lock so we could set up npm ci if we wanted to

Reply:
happy to remove it though, if we're not ready to switch over to that yet